src/shared/managers/SubscriptionManager.ts

```diff
@@ -188,20 +186,35 @@ export class SubscriptionManager {
     }
   }
 
-  async updatePushSubscriptionNotificationTypes(notificationTypes: number): Promise<void> {
+  async updateNotificationTypes(): Promise<void> {
+    const notificationTypes = await this.getNotificationTypes()
+    await this.updatePushSubscriptionNotificationTypes(notificationTypes);
+  }
+
+  async getNotificationTypes(): Promise<SubscriptionStateKind> {
+    const { optedOut } = await Database.getSubscription();
+    if (optedOut) {
+      return SubscriptionStateKind.MutedByApi;
```

Comment:
Maybe we should think about renaming this since the notification type could be set from the SDK. Could we change it to 'Granted', 'Denied', and 'Default'?

Reply:
Renamed this as as well as the `SubscriptionStateKind`, `Default`, so this isn't misleading. Also added some comments in SubscriptionStateKind to further clarify in commit eb4f784912fedf3bab80e5c2a6547ec4646df204.